subnet/deployment-generator/scripts/add_node.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -eo pipefail
+
+FILE="../docker-compose.yml"
```

Comment:
suggest to also use `cd "$(dirname "$0")"` to make script able to run from any directory